src/ZiraLink.Api.Application/Tools/ITools.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ZiraLink.Api.Application.Tools
+{
+    public interface ITools
+    {
+        Task<bool> CheckDomainExists(string domainUrl);
```

Comment:
Better to extract Ping method from CheckDomainExistsd

Reply:
Done.